README.md

```diff
@@ -99,7 +99,7 @@ Imagine that you take the code from this question, save it to a file called `sus
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+The output would be "sushi is tasty" because we set howMuchILikeSushi equal to 2, and 2 is greater than 0.
```

Comment:
Correct! Although `x` is still undefined, the line that would be evaluating it never gets reached.